integration-tests/capacity/replenishment.test.ts

```diff
@@ -149,3 +139,18 @@ describe("Capacity Replenishment Testing: ", function () {
     })
   })
 });
+
+async function drainCapacity(call, stakeProviderId: u64, stakeKeys: KeyringPair): Promise<bigint> {
+  const totalCapacity = (await getRemainingCapacity(stakeProviderId)).toBigInt();
```

Comment:
Small nit; this line could be moved over two spaces

Reply:
Fixed.